lib/chef/resource/chocolatey_source.rb

```diff
@@ -78,6 +86,26 @@ def fetch_source_element(id)
         end
       end
 
+      action :enable do
```

Comment:
can you add introduced "14.6" inside these actions as well so our documentation will pick up when we added this

Reply:
What is the proper syntax for introduced under an action?
```
      action :enable do
        introduced "14.6",
        description "Enables a Chocolatey source."
```

or

```
      action :disable do
        introduced: "14.6",
        description "Disables a Chocolatey source."
```?